### ISSUE TITLE
Fine-Tune 

### ISSUE THREAD

**Issue by JamesWandy** [LiheYoung/Depth-Anything#5]
Will the code for fine-tuning the models be released? 
Thank you for your excellent work.

**Comment by LiheYoung** [comment]
Do you mean fine-tuning for in-domain and zero-shot metric depth estimation? We have released the training code [here](https://github.com/LiheYoung/Depth-Anything/tree/main/metric_depth).

**Comment by VimalMollyn** [comment]
Thanks for the great work! Do you plan to release the pretraining code as well?

**Comment by JamesWandy** [comment]
@LiheYoung No I'm not interested in metric depth estimation. I mean fine tuning on new data to achieve better relative depth estimation.

**Comment by sysuyl** [comment]
hello, I also intertested in semantic segmentation. Will the code for fine-tuning segmentation be released. @LiheYoung

**Comment by adam99goat** [comment]
Hi, I am wondering whether it is possible to fine-tune on custom dataset for relative depth estimation as well.

**Comment by jorgejgnz** [comment]
I am also interested in fine-tuning this model with a custom dataset

**Comment by cht619** [comment]
How to use the model in semantic segmentation?

**Comment by rafaelagrc** [comment]
Yes, I am also interested in finetuning for relative depth estimation

**Comment by shilpaullas97** [comment]
@adam99goat , @jorgejgnz , @rafaelagrc 

Did you get a chance to fine-tune the model for relative depth estimation?
Also, is there any way to fine tune it for custom unlabeled dataset?
